src/docfx/lib/schema/JsonSchemaValidator.cs

```diff
@@ -747,10 +747,13 @@ private Error GetError(JsonSchema schema, Error error)
             }
 
             if (!string.IsNullOrEmpty(error.PropertyPath) &&
-                schema.Rules.TryGetValue(error.PropertyPath, out var attributeCustomRules) &&
+                schema.Rules.TryGetValue(error.PropertyPath, out var attributeCustomRules) && // todo remove?
```

Comment:
why not handle these error mapping in ErrorLog, just like JsonSchemaTransformer?

Reply:
`schema.Rules` are saved in JsonSchema and currrent was decoupled yet. We need move `schema.Rules` to `JsonSchemaValidatorExtension`

Reply:
comments changed.